tests/Aspire.Hosting.Tests/Utils/CommandLineArgumentsEvaluator.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Tests.Utils;
+
+public static class CommandLineArgumentsEvaluator
```

Comment:
Can we combine this with 

https://github.com/dotnet/aspire/blob/e516d8250ab4a507f03148b894e4ae0c5c398751/tests/Aspire.Hosting.Tests/Utils/ArgumentEvaluator.cs#L6-L8

It looks duplicated.